src/sinol_make/commands/run/__init__.py

```diff
@@ -1152,14 +1152,9 @@ def run(self, args):
         for solution in solutions:
             lang = package_util.get_file_lang(solution)
             for test in self.tests:
-                try:
-                    package_util.get_time_limit(test, self.config, lang, self.args)
-                except KeyError:
-                    util.exit_with_error(f'Time limit was not defined for test {os.path.basename(test)} in config.yml.')
-                try:
-                    package_util.get_memory_limit(test, self.config, lang, self.args)
-                except KeyError:
-                    util.exit_with_error(f'Memory limit was not defined for test {os.path.basename(test)} in config.yml.')
+                # The functions will exit if the limits are not set
+                package_util.get_time_limit(test, self.config, lang, self.args)
+                package_util.get_memory_limit(test, self.config, lang, self.args)
```

Comment:
```suggestion
                _ = package_util.get_time_limit(test, self.config, lang, self.args)
                _ = package_util.get_memory_limit(test, self.config, lang, self.args)
```
Let's make it clear that the function returns a value and we are ignoring it. (I'm unsure whether the code I wrote works)